keras/src/layers/merging/concatenate.py

```diff
@@ -145,12 +145,13 @@ def compute_mask(self, inputs, mask=None):
                 # Input is unmasked. Append all 1s to masks,
                 masks.append(ops.ones_like(input_i, dtype="bool"))
             elif mask_i.ndim < input_i.ndim:
-                # Mask is smaller than the input, expand it
-                masks.append(
-                    ops.broadcast_to(
-                        ops.expand_dims(mask_i, axis=-1), ops.shape(input_i)
-                    )
+                # Broadcast mask shape to match in a way where we capture the
+                # input as a symbolic input in the op graph.
+                mask_i = ops.logical_or(
```

Comment:
So you're doing an `ops.logical_or` just to do the broadcast.

Do you understand why this works and `ops.broadcast_to` doesn't?

In other words, should we try to fix `ops.broadcast_to`? Is it a bug in `tf.broadcast_to`?